test/index.html

```diff
@@ -16,7 +16,8 @@
   <script>
     WCT.loadSuites([
       'basic.html',
-      'basic.html?dom=shadow'
+      'basic.html?dom=shadow',
+      'basic.html?dom=shadow&lazyRegister=true&useNativeCSSProperties=true'
```

Comment:
The whole set of these seems necessary to get Polymer to use native CSS custom properties: https://github.com/Polymer/polymer/blob/ff6e884ef4f309d41491333860a8bc9c2f178696/src/lib/settings.html#L55.